src/queries/get-events.ts

```diff
@@ -0,0 +1,35 @@
+export const GET_EVENTS = `
+  query (
+    $input: ConnectionInput
+    $filter: SearchConnectionFilter!
+    $sort: KeywordSort
+  ) {
+    keywordSearch(input: $input, filter: $filter, sort: $sort) {
+      count
+      edges {
+        node {
+          id
+          result {
+            ... on Event {
+              id
```

Comment:
Not 100% sure (and might be wrong) but it is possible to create a fragment for Event and replace this Event object to it?

Reply:
I've never thought of it as the doc says we need inline fragment for querying union types (ref: https://graphql.org/learn/queries/#inline-fragments). But I found actually both fragment and inline fragment work for it. (ref: https://www.apollographql.com/docs/ios/fetching/type-conditions/).

```
export const GET_EVENTS = `
  query (
    $input: ConnectionInput
    $filter: SearchConnectionFilter!
    $sort: KeywordSort
  ) {
    keywordSearch(input: $input, filter: $filter, sort: $sort) {
      count
      edges {
        node {
          id
          result {
            ...ResultEvent
          }
        }
      }
    }
  }
  fragment ResultEvent on Event {
    id
    title
    eventUrl
    description
    dateTime
    venue {
      id
      name
      address
      lat
      lng
    }
    image {
      baseUrl
    }
  }
`;

```

Reply:
@Lada496 Thank you for sharing two references regarding regarding fragment and inline fragment! (Actually, I totally didn't know about inline fragment until you mentioned about it, lol) 

Imo, we can keep this inline fragment not to add extra task creating fragment.